client/src/login/loginReducer.js

```diff
@@ -0,0 +1,16 @@
+import { Map } from 'immutable';
+
+export const SAVE_USER = 'SAVE_USER';
```

Comment:
How about move it to a constants file then? That's the standard as soon as you get past a very simple app and we would have moved that direction anyway.